### DISCUSSION TITLE
Unauthorized for Listings API

### DISCUSSION THREAD

**Discussion by DearRude** [amzn/selling-partner-api-models#4074] [Q&A]
I am trying to use the `/listings/2021-08-01/items/{sellerId}/{sku}` endpoint to get details on my items. I have successfully done so for ~7000 items in the Canadian marketplace and ~400 items in the United States marketplace.

However, when I am fetching information on about ~30 items in the United States marketplace, I get the following error:

```
Code: Unauthorized Message: Access to requested resource is denied.
```
Any idea how to troubleshoot this problem?

**Comment by chapmanjw** [comment]
@DearRude The most likely scenario is that the `sellerId` value being used doesn't match the authorized account. However, to investigate your specific requests further, can you please open a developer support ticket? https://developer.amazonservices.com/support Including the request details, timestamps, and request ids would help us to see what specifically happened on these requests.

**Reply by DearRude** [reply]
I think the specified `sellerId` is not recognized correctly since the id of two sellers I want fetch data from have the **same seller id**: 
[TopSelectCanada](https://www.amazon.ca/sp?seller=A2U7BYCM1HXLY3) and [QuickWare](https://www.amazon.com/sp?ie=UTF8&seller=A2U7BYCM1HXLY3)

I don't know how to distict them in the API call.
